x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_eql_search_request.ts

```diff
@@ -87,6 +87,10 @@ export const buildEqlSearchRequest = ({
           filter: requestFilter,
         },
       },
+      // the allow_partial$ query parameters supersede xpack settings on cluster
```

Comment:
Typo?

Reply:
Originally there were two properties we could send in the request, `allow_partial_serach_results` and `allow_partial_sequence_results` and I tried to be "smart" writing that comment but obviously not relevant since we are not enabling this for sequence queries. I can update this to just reference the allow_partial_search_results param without the `$`